src/lib.rs

```diff
@@ -16,19 +16,22 @@ use microkelvin::{
 };
 
 #[derive(Clone, Canon, Debug)]
+/// Internal structure to be used as tuple in the map buckets
 pub struct KvPair<K, V>(K, V);
 
 #[derive(Clone, Canon, Debug)]
 enum Bucket<K, V, A> {
     Empty,
     Leaf(KvPair<K, V>),
-    Node(Annotated<Hamt<K, V, A>, A>),
+    Node(Annotated<KelvinHamt<K, V, A>, A>),
 }
 
 #[derive(Clone, Canon, Debug)]
-pub struct Hamt<K, V, A>([Bucket<K, V, A>; 4]);
+pub struct KelvinHamt<K, V, A>([Bucket<K, V, A>; 4]);
 
-impl<K, V, A> Compound<A> for Hamt<K, V, A>
+pub type Hamt<K, V> = KelvinHamt<K, V, ()>;
```

Comment:
Maybe add some doc comments since the type is exported.

Reply:
Agreed, but its out the scope of the PR. Issue added to keep track:

https://github.com/dusk-network/dusk-hamt/issues/8